distribution/bin/enso

```diff
@@ -1,5 +1,5 @@
 COMP_PATH=$(dirname "$0")/../component
 
-JAVA_OPTS="--add-opens=java.base/java.nio=ALL-UNNAMED $JAVA_OPTS"
+JAVA_OPTS="--add-opens=java.base/java.nio=ALL-UNNAMED -Xss16M $JAVA_OPTS"
```

Comment:
This does not affect the `ensoup` launcher nor engine that is started through project manager.

To update those, please add a flag in `distribution/manifest.template.yaml`.

Reply:
Done